<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bt_gohome">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>740</y>
     <width>135</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>500</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_rental">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>740</y>
     <width>135</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>대여</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_return">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>740</y>
     <width>135</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>반납</string>
   </property>
  </widget>
  <widget class="QComboBox" name="book_item">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>135</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>도서명</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>저자</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="bt_Search">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>30</y>
     <width>135</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>1131</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>730</y>
     <width>311</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
